charts/monitoring/kube-state-metrics/templates/deployment.yaml

```diff
@@ -42,17 +43,16 @@ spec:
       containers:
       - name: kube-state-metrics
         image: '{{ .Values.kubeStateMetrics.image.repository }}:{{ .Values.kubeStateMetrics.image.tag }}'
-        # Ref: https://kubernetes.io/blog/2021/04/13/kube-state-metrics-v-2-0/#what-is-new-in-v2-0
-        # Issue and solutions: https://github.com/kubernetes/kube-state-metrics/issues/1501#issuecomment-863020915
-        # and https://github.com/kubernetes/kube-state-metrics/issues/1489#issuecomment-851970288
         args:
-        - --metric-labels-allowlist=pods=[app.kubernetes.io/name,app.kubernetes.io/component,app.kubernetes.io/instance,component,part-of,app,unit],deployments=[app.kubernetes.io/name,app.kubernetes.io/component,app.kubernetes.io/instance]
+        - --metric-labels-allowlist=pods=[app.kubernetes.io/name,app.kubernetes.io/component,app.kubernetes.io/instance,component,part-of,app,unit],deployments=[app.kubernetes.io/name,app.kubernetes.io/component,app.kubernetes.io/instance],persistentvolumeclaims=[excluded-from-alerts]
         {{- if .Values.kubeStateMetrics.customResourceState.enabled }}
         - --custom-resource-state-config-file=/etc/customresourcestate/config.yaml
         volumeMounts:
         - name: customresourcestate-config
           mountPath: /etc/customresourcestate
           readOnly: true
+        - name: tmp
```

Comment:
should this volume be present within the `if` for customresourcestate?

Reply:
Oh great catch! No it shouldn't.

Reply:
Fixed :)